prow/cmd/tide/config.md

```diff
@@ -15,6 +15,12 @@ The following configuration fields are available:
 * `merge_method`: A key/value pair of an `org/repo` as the key and merge method to override
    the default method of merge as value. Valid options are `squash`, `rebase`, and `merge`.
    Defaults to `merge`.
+* `merge_approve_tag`: The tag appended to the automatic commit
```

Comment:
same here, I don't think this is particularly clear that unset / empty string -> feature is disabled

I think it's also worth noting what is considered for approval, it's possible to use tide without using our approval mechanisms since tide just cares about matching queries and the plugin system for label management is extensible (or labels could even be managed by humans or other tools or...)

Reply:
> same here, I don't think this is particularly clear that unset / empty string -> feature is disabled

Done in b0c751d -> 97f6fa5.

> I think it's also worth noting what is considered for approval...

Lots more on this line in [this thread][1] ;).  Let's keep the rest of that discussion there.

[1]: https://github.com/kubernetes/test-infra/pull/9400#discussion_r218105016